external-crates/move/crates/move-package-alt/src/dependency/dependency_set.rs

```diff
@@ -57,6 +67,18 @@ impl<T> DependencySet<T> {
         result
     }
 
+    /// Convenience method to return either [default_deps] or [deps_for_env] depending on [env]
+    pub fn deps_for(&self, env: Option<&EnvironmentName>) -> BTreeMap<PackageName, &T> {
+        match env {
+            Some(env) => self.deps_for_env(env),
+            None => self
+                .default_deps()
+                .iter()
+                .map(|(pkg, dep)| (pkg.clone(), dep))
```

Comment:
Would it be better to do
```suggestion
                .into_iter()
                .map(|(pkg, dep)| (pkg, dep))
```

Reply:
I'm not sure the best thing here - if we make this change then `dep` is also an owned value, but we need to return a reference, so I think we only want to clone the keys, but not the values.